test/Fcs_load_and_transform.jl

```diff
@@ -7,29 +7,48 @@ as this function is only the basic parsing of the binary
 FCS, we need to see what functionality is missing and
 extend this in the original package
 =#
-
 using Distributed
 
 # p = addprocs(2)
 #
 # @everywhere using GigaSOM
 
+
 using GigaSOM
```

Comment:
you may write: `using GigaSOM, DataFrames, XLSX, CSV`

Reply:
done